packages/components/psammead-brand/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 3.0.1   | [PR#609](https://github.com/bbc/psammead/pull/609) Add lang attribute to brand component |
```

Comment:
This is a breaking change since we are replacing `brandName` with `product`. You should bump it to a major version.

Reply:
@sadickisaac so this should be version 4.0.0?

Reply:
Yes.